crowd-ui/src/features/translationValidation/components/main/Main.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useState, useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { FormattedMessage } from 'react-intl';
+import { getSentenceFetch, postUserData } from '../../action';
+import './main.scss';
+import { selectSentences, selectLoading } from '../../slice';
+
+export default function Main() {
+  enum answerType {
+    Yes = 1,
+    No = 2,
+    NonSelect = 3,
+  }
+  const [userAnswer, setUserAnswer] = useState<number>(answerType.NonSelect);
+  const sentence: any = useSelector<any, any>(selectSentences);
+  const isLoading: any = useSelector<any, any>(selectLoading);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    if (isLoading) {
+      dispatch(getSentenceFetch());
+    }
+  }, [isLoading]);
+
+  let mainSentence = '';
+  let translateSentence = '';
```

Comment:
what's that for?

Reply:
it's not for. it's if 
if my api failed from backend, my app is working without error